### ISSUE TITLE
Do we need all those gmail related libs ?

### ISSUE THREAD

**Issue by vshivam** [privly/privly-android#46]
@gitanshu We have these dependencies for gmail but do we really need all of these ?

    compile 'com.google.http-client:google-http-client-gson:1.19.0'
    compile 'com.google.http-client:google-http-client-jackson2:1.19.0'
    compile 'com.google.android.gms:play-services:+'
    compile 'com.google.api-client:google-api-client:1.19.0'
    compile 'com.google.api-client:google-api-client-jackson2:1.19.1'
    compile 'com.google.api-client:google-api-client-android:1.19.1'
    compile 'com.google.api-client:google-api-client-gson:1.19.1'
    compile 'org.apache.commons:commons-lang3:3.3.2'
    compile 'com.google.apis:google-api-services-gmail:v1-rev9-1.19.0'
    compile 'com.fasterxml.jackson.core:jackson-core:2.5.0'
    compile 'com.google.oauth-client:google-oauth-client:1.19.0'
    compile 'com.google.http-client:google-http-client-android:1.19.0'
    compile 'com.google.http-client:google-http-client:1.19.0'
    compile 'org.apache.httpcomponents:httpcore:4.4'

For me the app seems to work with these two dependencies : 

    compile 'com.google.android.gms:play-services:+'
    compile 'com.google.apis:google-api-services-gmail:v1-rev24-1.19.1'

**Comment by gitanshu** [comment]
When I moved the project to gradle, I replaced each jar file with its respective gradle dependency counterpart. Gmail API does need all these libraries. However, looks like gradle's transitive dependency management does the job for us if we only mention the google-api-services-gmail gradle artifact. Here is the gradle dependency tree with your suggested configuration. This explains everything: 

![gradle-dependencies](https://cloud.githubusercontent.com/assets/1853790/6537927/9b3d69dc-c481-11e4-8cec-e7dc2b796f7c.PNG)

So it's safe to remove the redundant dependencies.
